sdk/nodejs/appservice/zMixins.ts

```diff
@@ -423,15 +569,24 @@ export class CallbackFunctionApp<C extends Context<R>, E, R extends Result> exte
  * azure resource an an FunctionApp that will be triggered when something happens to that resource.
  */
 export abstract class EventSubscription<C extends Context<R>, E, R extends Result> extends pulumi.ComponentResource {
-    public readonly functionApp: CallbackFunctionApp<C, E, R>;
+    public readonly functionApp: MultiFunctionApp;
 
     constructor(type: string, name: string,
                 bindings: pulumi.Input<BindingDefinition[]>,
                 args: CallbackFunctionAppArgs<C, E, R>,
                 opts: pulumi.ComponentResourceOptions = {}) {
         super(type, name, undefined, opts);
 
-        this.functionApp = new CallbackFunctionApp(name, bindings, args, { parent: this });
+        const multiArgs = {
+            ...args,
+            functions: [{
+                name,
+                bindings,
+                body: serializeFunctionCallback(args),
+                route: name, // this is not entirely correct, will be fixed with a specific implementation of HttpFunction

```

Comment:
not certain i understand this comment.

Reply:
The route of HTTP functions depends on their configuration - you can check the other PR's `HttpFunction` class which passes this parameter down. I can't see how to implement this without making `HttpFunction` in this PR.